blocks/columns/columns.css

```diff
@@ -99,6 +111,20 @@ div.slim-promo a {
   text-decoration: underline;
 }
 
+.category-grid {
+  border-bottom: 0.1rem solid #dcdde6;
+  padding-bottom: 60px;
+}
+
+.category-grid h4 {
+  font-family: var(--fixed-font-family);
+  text-transform: uppercase;
+  font-size: 1.4rem;
+  font-weight: 600;
+  letter-spacing: .075rem;
+  line-height: 2rem;
+}
+
```

Comment:
I think it should be top-aligned, but it is not. You could try: 
.columns.category-grid > div {
  align-items: flex-start;
}

Reply:
I didn't see this issue, but I added it anyway.

Reply:
I still see it 
![Screenshot 2023-06-21 at 14 40 55](https://github.com/hlxsites/Vonage/assets/2250964/77b10d7a-fbf2-4f7d-b223-995d76819bd9)

Reply:
you may have jumped the gun and previewed before I pushed up the change:

<img width="1073" alt="image" src="https://github.com/hlxsites/Vonage/assets/171559/f2c1f0f9-3f08-4db1-9053-bade21b5cba5">